### PR TITLE
Comarch connector

### PR DESCRIPTION
**Pull request by ewa-git**
please review my code

### CODE REVIEW

**Review by KrzKos — APPROVED** [review]
I've checked only HttpUtils. Works fine!

File: fakturogen-invoice/src/main/java/pl/fakturogen/invoice/dao/repository/InvoiceRepository.java [hunk 1]
```diff
@@ -1,7 +1,4 @@
 package pl.fakturogen.invoice.dao.repository;
 
-import org.springframework.data.jpa.repository.JpaRepository;
-import pl.fakturogen.invoice.dao.entity.Invoice;
-
-public interface InvoiceRepository extends JpaRepository<Invoice, Long> {
+public class InvoiceRepository {
```
Comment (rowinskidamian) [inline fakturogen-invoice/src/main/java/pl/fakturogen/invoice/dao/repository/InvoiceRepository.java:3]: we should have here previous version with extends JpaRepository, version with current change is step backwards

File: fakturogen-invoice/src/main/java/pl/fakturogen/invoice/dao/repository/InvoiceTemplateRepository.java [hunk 2]
```diff
@@ -1,7 +1,4 @@
 package pl.fakturogen.invoice.dao.repository;
 
-import org.springframework.data.jpa.repository.JpaRepository;
-import pl.fakturogen.invoice.dao.entity.InvoiceTemplate;
-
-public interface InvoiceTemplateRepository extends JpaRepository<InvoiceTemplate, Long> {
+public class InvoiceTemplateRepository {
```
Comment (rowinskidamian) [inline fakturogen-invoice/src/main/java/pl/fakturogen/invoice/dao/repository/InvoiceTemplateRepository.java:3]: we should have here previous version extends JPARepository

File: fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/Mapper.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package pl.fakturogen.comarch.connector.connectors;
```
Comment (rowinskidamian) [inline fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/Mapper.java:1]: we need to think if we need this interface

File: fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/MapperImpl.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package pl.fakturogen.comarch.connector.connectors;
```
Comment (rowinskidamian) [inline fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/MapperImpl.java:1]: need to think if we need this impl

File: fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/downloader/DataDownloader.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package pl.fakturogen.comarch.downloader;
```
Comment (rowinskidamian) [inline fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/downloader/DataDownloader.java:1]: we can delete this class

Comment (rowinskidamian) [inline fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/downloader/DataDownloader.java:1]: we can delete package downloader

File: fakturogen-user/src/main/java/pl/fakturogen/user/dao/entity/UserExternalApi.java [hunk 6]
```diff
@@ -12,7 +12,7 @@
 @Entity
 @Table(name = UserExternalApi.TABLE_NAME)
 public class UserExternalApi {
-    public static final String TABLE_NAME = "users_external_api";
+    public static final String TABLE_NAME = "users external api";
```
Comment (rowinskidamian) [inline fakturogen-user/src/main/java/pl/fakturogen/user/dao/entity/UserExternalApi.java:15]: I think that we should have previous version "users_external_api" current version doesn't have '_' between words

File: fakturogen-web/src/main/resources/application.properties [hunk 7]
```diff
@@ -1,3 +1,18 @@
+
+#spring.datasource.url=jdbc:h2:~/h2database/fakturogen2;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;AUTO_SERVER=TRUE;FILE_LOCK=SOCKET;
```
Comment (rowinskidamian) [inline fakturogen-web/src/main/resources/application.properties:2]: I think that line 1 to 15 we can delete, as these are only comments, and the same is in next lines

**Review by rowinskidamian — CHANGES_REQUESTED** [review]
few small changes are needed outside the comarch-connector module, classes in comarch-connector module perffecto! :)

**Comment by ewa-git** [comment]
please see my corrections

**Review by rowinskidamian — APPROVED** [review]
super :)

**Pull request merged**